flab_bada/domain/email/email_controller.py

```diff
@@ -0,0 +1,35 @@
+from fastapi import APIRouter, status, Depends, HTTPException
+from starlette.responses import JSONResponse
+from flab_bada.schemas.users import EmailSecret
+from flab_bada.domain.email.email_service import EmailService
+from flab_bada.logging.logging import log_config
+from flab_bada.schemas.users import EmailSchema
+
+
+log = log_config("email controller")
+
+
+email_router = APIRouter()
+
+
+@email_router.post("/send/email")
+async def simple_send(email: EmailSchema, email_service: EmailService = Depends()) -> JSONResponse:
```

Comment:
```suggestion
async def simple_send(email_schema: EmailSchema, email_service: EmailService = Depends()) -> JSONResponse:
```
변수명도 그 의미를 충분히 드러나도록 설정하는게 좋습니다 `email` 이라는 단어는 서비스일까요? 레포지토리일까요? 스키마일까요? 모호한 표현은 지양해봅시다